dags/iop/parser.py

```diff
@@ -75,3 +80,13 @@ def _get_related_article_doi(self, article):
         except AttributeError:
             self.logger.error("No related article dois found", dois=self.dois)
             return
+
+    def _get_arxiv_eprints(self, article):
+        arxivs_value = article.find(
+            "front/article-meta/custom-meta-group/custom-meta/meta-value"
+        )
+        try:
+            pattern = re.compile(r"(arxiv:|v[0-9]$)", flags=re.I)
+            return [{"value": pattern.sub("", arxivs_value.text.lower())}]
```

Comment:
I would prefer to create a variable with proper formatting of an eprint  (so this `pattern.sub("", arxivs_value.text.lower())`) and then return, it will be more readable